demo/lightning/text-based/README.md

```diff
@@ -55,3 +55,33 @@ kubectl apply -f deploy.yaml
 ```
 
 *Note: the example YAML file  and model code assumes that you have jobset and Kueue enabled on your GKE cluster. For easy compatability we recommend creating a cluster with [XPK](https://github.com/google/xpk) which will configure these features automatically.*
+
+## Benchmark Results
+The tables below show the benchmark results from our internal testing. All testing was performed on a 500 node CPU only cluster, backed by a bucket co-located in the same region (this is critical for maximizing dataloading performance). The test cluster was comprised of 512 [n2-standard-64](https://cloud.google.com/compute/docs/general-purpose-machines#n2_machine_types) nodes. The executions in the benchmarking effort features the following shared parameters:
+
+- num_workers: 1
+- batch_size: 512
+
+All benchmarks were run against sets of parquet files from the [Hugging Face Fineweb](https://huggingface.co/datasets/HuggingFaceFW/fineweb) dataset. Our benchmark code only divides to file-level, so maximum parallelization is limited to total file count.
+
+This code is still being updated and improved, so all benchmark results are subject to change.
+
+### Basic Throughput Testing
+For this testing, batch size was limited to 2000, thus time metrics were not comprable and only throughput was measured.
```

Comment:
This sentence doesn't really make it clear to me what the difference is between this test and the following one. Can you expand on it a bit? Does this mean we only iterated to 2000 objects (i.e. like 4 batches)?

Reply:
2000 batches, i'll clarify.